src/main/resources/avro/metadata.avdl

```diff
@@ -147,6 +148,8 @@ record Sample {
   /**
   The time at which this sample was taken from the individual, in milliseconds
   from the epoch.
+  TODO: should we consider using relative time (e.g. to the start of a study) rather than absolute time?
```

Comment:
The reason I proposed to use a relative time is that in practice, you almost never have exact dates/times available in a research context. So this field will be left null, or people will try to encode relative times in absolute times, so that you get workarounds such as dates around Jan 1, 1900, which is not desirable. Only in a medical context (hospital) you can expect actual sample dates to be available.
But it's always difficult to balance theoretical correctness and tidiness of the data model vs applicability in practice, since I'm new to the group I'm not sure what reasoning should be applied.